src/components/Accordion/index.js

```diff
@@ -39,19 +33,20 @@ export default class Accordion extends Component {
     }
 
     render() {
-        const { id, children, style, multiple } = this.props;
+        const { id, children, style, multiple, className } = this.props;
         const { activeNames } = this.state;
+        const context = {
```

Comment:
sorry let's put this in state because there is not need to recreate this object in each rerender:
```
this.state = {
    activeNames: props.activeSectionNames,
    multiple: props.multiple,
    privateOnToggleSection: this.handleToggleSection,
};
```
but be careful now:
```this.handleToggleSection = this.handleToggleSection.bind(this);```
must be above the state
then use:
```<Provider value={this.state}>```